packages/core-manager/src/actions/info-core-version.ts

```diff
@@ -24,15 +29,7 @@ export class Action implements Actions.Action {
     }
 
     private getChannel(): string {
-        const channels: string[] = ["next"];
-
-        let channel = "latest";
-        for (const item of channels) {
-            if (this.app.version().includes(`-${item}`)) {
-                channel = item;
-            }
-        }
-
-        return channel;
+        const config = this.cli.resolve(Cli.Services.Config);
+        return config.get("channel");
```

Comment:
```suggestion
        return this.cli.resolve(Cli.Services.Config).get("channel");
```

Unnecessary temporary assignment.